.github/workflows/ci-playwright.yml

```diff
@@ -0,0 +1,57 @@
+name: CI - Playwright
```

Comment:
Please rename to "CI - FAST Testing" to be consistent with other naming patterns used on GitHub Actions

Reply:
cool